.github/workflows/publish_charm.yaml

```diff
@@ -61,6 +68,7 @@ jobs:
       - name: Select charmhub channel
         uses: canonical/charming-actions/channel@2.6.3
         id: select-channel
+        continue-on-error: ${{ inputs.select-channel-continue-on-error }}
```

Comment:
I think we can just set this to true and always fallback. Note that there's already a parameter to to pass the destination channel, so I'd say not having this set to true is actually a bug. WDYT?

Reply:
Yes, we are currently using the `channel` parameter [here](https://github.com/canonical/k8s-operator/blob/main/.github/workflows/publish-charms.yaml#L55)
I agree with you, it's a bug that it's not set to `true`. I'll update my PR to align with your suggestion. Thanks for the review @arturo-seijas!